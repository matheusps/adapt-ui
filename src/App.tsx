import React from 'react'

const App = () => {
  return <h1>React Stellar</h1>
}

export default App
