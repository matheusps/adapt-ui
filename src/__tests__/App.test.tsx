import React from 'react'
import { render } from 'react-testing-library'
import App from '../App'

describe('App component', () => {
  it('should render', () => {
    const component = render(<App />)
    expect(component).toBeTruthy()
  })

  it('should match snapshot', () => {
    const component = render(<App />)
    expect(component.asFragment()).toMatchSnapshot()
  })
})
