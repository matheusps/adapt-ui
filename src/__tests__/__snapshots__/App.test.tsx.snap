// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should match snapshot 1`] = `
<DocumentFragment>
  <h1>
    React Quarks
  </h1>
</DocumentFragment>
`;
